style.css

```diff
@@ -0,0 +1,91 @@
+body {
+  background-color: #1c1a19;
+  margin: 0;
+  color: #fff;
+  width: 375px;
+  height: 590px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
+  top: 56px;
+}
+
+.container {
+  margin: 0 24px;
+  overflow: hidden;
+}
+
+.header {
+  background-color: #3e3c3c;
+  margin: 0;
+  height: 92px;
+  width: 100%;
+}
+
+.main h1 {
+  font-family: 'Crete Round', serif;
+  color: #ff6800;
+  font-weight: 400;
+  font-size: 3.5rem;
+  line-height: 4rem;
+  letter-spacing: 0.37px;
+  margin: 12px 0;
+}
+
+.text-subheading h3 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  margin: 12px 0;
+  font-size: 1.5rem;
+}
+
+.text-body p {
+  font-family: 'Poppins', sans-serif;
+  line-height: 32px;
+  font-weight: 400;
+  font-size: 1rem;
+  margin: 12px 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  margin-top: 120px;
+}
+
+.navbar p {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 1.25rem;
+  line-height: 1.5rem;
+  margin: 0;
+}
+
+.socials {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 37%;
+  padding-left: 3%;
+  margin-bottom: 32px;
+}
+
+.linkedin {
+  display: flex;
+  flex-direction: column;
+  margin-right: 4px;
+}
+
+.flex-confused {
+  display: flex;
+}
+
+.hamburger {
+  border: solid 2px #fff;
+  margin-bottom: 4px;
+  width: 20px;
+  border-radius: 5px;
+}
```

Comment:
- [ ] Kindly fix the bug generated at the mobile view, at the mobile screen the hamburger menu is no more.
### **See the Error**
![errorm](https://user-images.githubusercontent.com/55596970/158931688-6ed6ed96-e40d-4e50-9c9f-d24c02ddfe04.png)

Reply:
Please can you tell me the browser you are using? Because I have tested on Google chrome, firefox, and Microsoft Edge and the page seems to come out okay.


![image](https://user-images.githubusercontent.com/78151836/159105849-ea2183fd-c172-407d-aa8d-cccf230a2181.png)

Reply:
Hi @wale-prog 
I use the chrome browser

Reply:
Let me do it again

Reply:
Sorry, I can see it working fine now